androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/CalculatedTeamInMatchData.java

```diff
@@ -48,5 +46,13 @@
     public Integer totalNumScaleFoul;
     public Integer numSpilledCubesAuto;
     public Integer numSpilledCubesTele;
+    public Integer rankAgility;
+    public Integer rankDefense;
+    public Integer rankSpeed;
+    public Object startingPosition;
+    public String scoutName;
```

Comment:
Why are these added when they should be removed? Everything game specific (etc "cubes", "scale") should be removed for the following year's template